cosmoz-tabs.js

```diff
@@ -21,6 +21,13 @@ class CosmozTabs extends Polymer.mixinBehaviors([Cosmoz.TabbableBehavior], Polym
 				type: String,
 				value: 'name'
 			},
+			/**
+			 * If true, scroll buttons (left/right arrow) will be hidden for scrollable tabs.
+			 */
+			hideScrollButtons: {
+				type: Boolean,
+				value: true
```

Comment:
A bool with default value true can not be set with attributes?

Reply:
I sat the default to true because if scrollable were to be used, it would avoid having to remember to set both attributes. I thought the use case for having the scrollbuttons were slim.
But maybe this just confuses.